src/utils/daos_hdlr.c

```diff
@@ -1187,12 +1204,21 @@ cont_get_prop_hdlr(struct cmd_args_s *ap)
 		D_GOTO(err_out, rc);
 	}
 
+	/* Fetch the ACL separately in case user doesn't have access */
+	rc = daos_cont_get_acl(ap->cont, &prop_acl, NULL);
+	if (rc && rc != -DER_NO_PERM) {
+		fprintf(stderr, "failed to query container ACL "DF_UUIDF
+			": %s (%d)\n", DP_UUID(ap->c_uuid), d_errdesc(rc), rc);
+		D_GOTO(err_out, rc);
```

Comment:
I am just wondering whether we should still print the rest even if we couldn't fetch the ACL ...

Reply:
Kris is out this week, but I suspect that her thinking here was that if we get an error other than -DER_NO_PERM that things are probably in a bad state and it would be better to exit early.